rhoshift/main.py

```
@@ -1,128 +1,181 @@
                 return 1
-                
+
         except Exception as install_error:
-            installation_time = time.time() - installation_start_time if installation_start_time else 0
-            logger.error(f"❌ Installation failed with unexpected error: {install_error}")
+            installation_time = (
+                time.time() - installation_start_time if installation_start_time else 0
+            )
+            logger.error(
+                f"❌ Installation failed with unexpected error: {install_error}"
+            )
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)